.gitignore

```diff
@@ -70,5 +70,8 @@
 /c++/m4/lt~obsolete.m4
 /c++/samples/addressbook
 
+#cmake artefacts
```

Comment:
KDevelop defaults to `/c++/build`  folder. I was under false impression that this is cmake practice. But I just checked that qtcreator defaults to `/build-c++-Desktop-${CMAKE_BUILD_TYPE}. Dropping.